app/src/main/java/com/nervousfish/nervousfish/side_activities/LoginActivity.java

```diff
@@ -1,11 +1,13 @@
-package com.nervousfish.nervousfish;
+package com.nervousfish.nervousfish.side_activities;
```

Comment:
Wouldn't it be possible to create a different entry class which can be in the same package as the ServiceLocator? I'm not sure how this works, but I imagine it is possible...